src/amo/components/GuidesAddonCard/index.js

```diff
@@ -104,7 +88,7 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
 
     return addon ? (
       <Card>
-        {this.renderInstallError()}
+        <AddonInstallError i18n={i18n} error={this.props.installError} />
```

Comment:
You should not pass `i18n` here, it is injected in the `AddonInstallError` internally (via the `translate()` HOC).

Reply:
yep make sense